pkg/publisher/publisher.go

```diff
@@ -73,6 +76,30 @@ func FindIface(ip net.IP) (iface net.Interface, err error) {
 	return iface, fmt.Errorf("Couldn't find interface with IP address %s", ip)
 }
 
+// If an interface changes after we've started listening on it (say because it
+// got bridged), we need to be restarted to listen on the new interface that
+// contains the specified IP.
```

Comment:
I think that comments should be updated here(something like - verify that we join to mdns multicast on the interface holds the IP)

Reply:
I will update this.